prog/postgres/postgres_resource_nexus.rb

```diff
@@ -130,7 +139,7 @@ def before_run
       resource_id: postgres_resource.id,
       resource_name: postgres_resource.name,
       billing_rate_id: BillingRate.from_resource_properties("PostgresCores", "standard", postgres_resource.location)["id"],
-      amount: server.vm.cores
+      amount: representative_server.vm.cores
```

Comment:
So, we're not billing for HA yet? 💸

Reply:
I added that to a [separate PR](https://github.com/ubicloud/ubicloud/pull/1240) to keep this one simple.